docs/TerraformPlanResultsPR.md

```diff
@@ -0,0 +1,47 @@
+## [TerraformPlanResultsPR](../src/TerraformPlanResultsPR.groovy)
+
+Use this to post Terraform plan results in the comments of a PR.
+
+One-Time Setup:
+* Install the terraform-landscape gem on your Jenkins slaves. (ONLY when using `withLandscape(true)`)
+
+Requirements:
+* Enable CredentialsPlugin to set the GITHUB_TOKEN environment varaible. This is later used in the TerraformPlanResultsPR Plugin for authentication.
```

Comment:
typo: "varaible"

Reply:
Separately: I'm inclined to avoid saying specifically that  the CredentialsPlugin needs to be used.  You've written a plugin that accepts *any* environment variable, so I'd be inclined to let the User decide how that environment variable is created (with the intent of reducing dependencies between plugins).

Reply:
* typo has been fixed
* Updated docs and removed requirement for CredentialsPlugin